src/App.jsx

```diff
@@ -1,104 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentsById(postId) {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postId) || null
```

Comment:
```suggestion
    commentsFromServer.filter(comment => comment.postId === postId)
```
null is redundant here 'cause the filter method always returns the array